java/src/jmri/server/json/util/hello-client.json

```diff
@@ -4,15 +4,6 @@
     "type": "object",
     "description": "Schema data object in message from client to JMRI for type \"hello\"",
     "properties": {
-        "locale": {
```

Comment:
This is a breaking change, as clients are no longer able to report their preferred Locale. Revert deleting these properties, or explain (in detail) why clients should not be able to report preferred Locale.

Reply:
this removal was a mistake, thanks for catching it. reverted.